src/components/uploadPicture/uploadImage.tsx

```diff
@@ -135,9 +117,9 @@ const UploadImageComponent: React.FC<UploadImageComponentProps> = ({
       )}
       {currentStep === UploadStep.UploadPicture && (
         <div className={styles.uploadContainer}>
-          {capturedImage && (
+          {selectedFile && (
             <svg>
-              <image href={URL.createObjectURL(capturedImage)} />
+              <image href={URL.createObjectURL(selectedFile)} />
             </svg>
```

Comment:
Is there a reason this has to be an svg instead of just an Image tag? Not really sure if there's a reason to change it though.

Reply:
0825676